docs/pages/includes/config-reference/proxy-service.yaml

```diff
@@ -127,6 +127,7 @@ proxy_service:
     # Also set using the CLI command:
     # 'teleport configure --acme --acme-email=email@example.com \
     # --cluster-name=tele.example.com -o file'
+    # This should NOT be enabled in a highly available Teleport deployment
```

Comment:
Can we say why or link to more detailed docs?

Reply:
{"caller":"alpnproxy/proxy.go:337","component":"alpn:proxy","error":"429 urn:ietf:params:acme:error:rateLimited: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/failed-validation-limit/","level":"warning","message":"Failed to handle client connection."

this was the error encountered, let me add another line indicating it can lead to a lock when scaling behind a load balancer